src/tools/schema2kotlin.ts

```diff
@@ -182,6 +184,7 @@ ${imports.join('\n')}
     return `
 ${typeAliases.join(`\n`)}
 
+@Generated(value = "src/tools/schema2kotlin.ts")
```

Comment:
The generated files in g3 will not have the full script path location, but I'm not sure how we can achieve that, as we'd need different goldens for the tests in the two different scenarios.